Cargo.toml

```diff
@@ -10,5 +10,9 @@ repository = "https://github.com/manishearth/elsa"
 keywords = ["data-structure", "map", "frozen", "cache", "arena"]
 categories = ["data-structures", "caching"]
 
+[features]
+default = ["indexmap"]
```

Comment:
Should the feature be on by default, or not?

Reply:
No extra dependencies by default, please

Reply:
@Manishearth fixed